client/elife-theme/src/index.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable import/extensions */
 import './fonts/index.css'
 
+const gridUnit = 6
```

Comment:
Isn't there a better way to do this? I would assume that when `theme.space` is called somewhere in the code you have access to the entire theme object so you can defer the value of `theme.space` until execution (e.g. with a function), and then get `theme.gridUnit` somehow. I would have to test this, but I think something like ``css`
calc(${props => props.theme.gridUnit} * 2);` `` would work (with `css` from styled-components).

Reply:
Yes, that calculation works and is the expected way to use gridUnit. The `space` property is required by `grid-styled`.

Reply:
@dryhten @tamlyn So should I be removing `space` from the theme after all? Or are we leaving it in here so we can use `space[2]` in place of `verticalXs` (for example)?

I'm eager to get this merged in so that I can use the appropriate variable names while styling components :pray:

Reply:
`space` is required in order to set `grid-styled` to use our spacing scale (it defaults to powers of 2 otherwise). Whether we use `space[2]` instead of `calc` doesn't affect this PR.

Reply:
ok :+1:

@dryhten happy to approve & merge?

Reply:
So deferring the execution inside `theme.space` seems like overly complicated at this point. I guess this is the simpler way of doing it.